pages/product.html

```diff
@@ -0,0 +1,85 @@
+<!DOCTYPE html>
+<html lang="fr">
+  <head>
+    <meta charset="utf-8" />
+    <meta
+      name="description"
+      content="Orinoco site de vente en ligne d'ours en peluche"
+    />
+    <meta
+      name="viewport"
+      content="width=device-width, initial-scale=1.0, viewport-fit=cover"
+    />
+    <link rel="shortcut icon" type="image/png" href="favicon.jpg" />
+    <link
+      rel="stylesheet"
+      type="text/css"
+      href="../assets/dist/css/product.css"
+    />
+    <script
+      defer
+      src="https://kit.fontawesome.com/ca4c042e28.js"
+      crossorigin="anonymous"
+    ></script>
```

Comment:
Missing og and twitter

Reply:
<!-- Open Graph / Facebook -->
    <meta property="og:type" content="website" />
    <meta
      property="og:url"
      content="https://orinoco.michael-blondeau.fr/pages/product.html"
    />
    <meta property="og:title" content="Orinoco" />
    <meta
      property="og:description"
      content="Orinoco est un site e-commerce à thème."
    />
    <meta
      property="og:image"
      content="http://orinoco.michael-blondeau.fr/assets/img/logo.png"
    />
    <!-- Twitter -->
    <meta property="twitter:card" content="summary_large_image" />
    <meta
      property="twitter:url"
      content="https://orinoco.michael-blondeau.fr/pages/product.html"
    />
    <meta property="twitter:title" content="Orinoco" />
    <meta
      property="twitter:description"
      content="Orinoco est un site e-commerce à thème."
    />
    <meta
      property="twitter:image"
      content="http://orinoco.michael-blondeau.fr/assets/img/logo.png"
    />